ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -9,18 +9,55 @@
 # - Produce an EVA YAML
 # - Run EVA
 #-------------------------------------------------------------
-#--------------- User modified options below -----------------
-obtype=$1
-cycle=${2:-2021080100}
+
+# ==============================================================================
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 [-c cycle] [-x] [-s] [-h] instrument [cycle]"
+  echo
+  echo "  -c  cycle to run DEFAULT=2021080100"
+  echo "  -x  don't run eva DEFAULT=run eva"
+  echo "  -s  just produce eva stats plots DEFAULT=produce lots of plots"
+  echo "  -h  display this message and quit"
+  echo
+  exit 1
+}
+
+# ==============================================================================
+
+cycle=2021080100
+run_eva=YES
+eva_stats_only=NO
+ 
+while getopts "c:hsx" opt; do
+  case $opt in
+    c)
+      cycle=$OPTARG
+      ;;
+    x)
+      run_eva=NO 
+      ;;
+    s)
+      eva_stats_only=YES
+      ;;
+    h|\?|:)
+      usage
+      ;;
+  esac
+done
+shift $((OPTIND - 1))
 
 if [ $# -lt 1 ] || [ $# -gt 2 ]; then
```

Comment:
can you not do instrument, cycle, and an eva option? that would be 3?

Reply:
@CoryMartin-NOAA I got a little confused here.  But it should just be 
```
if [ $# -ne 1 ]; then
```

The `shift` takes care of the rest.

Unless I misunderstand what you are asking

Reply:
the "or greater than 2" part was what I was concerned about

Reply:
But I see what you're saying, I've never used this like this but it is basically treating the args differently. As long as it works, it's good with me.